ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -155,8 +153,18 @@ private[apiserver] object ApiPackageManagementService {
       managementServiceTimeout,
       engine,
       darReader,
+      submissionIdGenerator,
     )
 
+  private def augmentSubmissionId: String => v1.SubmissionId =
```

Comment:
```suggestion
  private def augmentSubmissionId: String => SubmissionId =
```

Reply:
This seems to be almost the same as the one in `ApiConfigManagementService`, and the `isEmpty` check seems benign. Can I recommend factoring it out into a shared location?

Reply:
Done